hl7/4.2/modules/ROOT/edits/hl7-connector-config-topics.adoc

```diff
@@ -0,0 +1,269 @@
+= HL7 EDI - Additional Configuration Information - Mule 4
+
+Use these options to:
```

Comment:
What options?
Consider providing a more robust introduction to the content of this topic set, since the title is boilerplate and not very precise.

Reply:
Fixed.